### PR TITLE
CEDS-1468: fix for document prodcued page and clean out old itemscach…

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**
…ing service.

### CODE REVIEW

File: app/services/mapping/declaration/GoodsItemQuantityBuilder.scala [hunk 1]
```diff
@@ -41,7 +40,7 @@ object GoodsItemQuantityBuilder {
 
   def build(implicit cacheMap: CacheMap): DeclarationGoodsItemQuantityType =
     cacheMap
-      .getEntry[Seq[GovernmentAgencyGoodsItem]](ExportsItemsCacheIds.itemsId)
+      .getEntry[Seq[GovernmentAgencyGoodsItem]]("exportItems")
```
Comment (PatrykRudnicki) [inline app/services/mapping/declaration/GoodsItemQuantityBuilder.scala]: Why this change? I think it, it's better to have value in val, to change this if necessary only in one place

File: app/services/mapping/governmentagencygoodsitem/AdditionalDocumentsBuilder.scala [hunk 2]
```diff
@@ -19,14 +19,23 @@ package services.mapping.governmentagencygoodsitem
...
+  WriteOff => WCOWriteOff
+}
+import wco.datamodel.wco.declaration_ds.dms._2.AdditionalDocumentEffectiveDateTimeType.{
+  DateTimeString => WCODateTimeString
+}
+import wco.datamodel.wco.declaration_ds.dms._2.{
+  AdditionalDocumentEffectiveDateTimeType,
+  SubmitterNameTextType,
+  WriteOffQuantityQuantityType,
+  _
```
Comment (PatrykRudnicki) [inline app/services/mapping/governmentagencygoodsitem/AdditionalDocumentsBuilder.scala]: If we're importing at the end everything, do we need to have three types specified before?

Comment (mattclark-zerogravit) [reply app/services/mapping/governmentagencygoodsitem/AdditionalDocumentsBuilder.scala]: This will be intelliJ doing its thing as I cleaned up the imports as I had unused ones.

File: app/services/mapping/declaration/GoodsItemQuantityBuilder.scala (hunk 1 above)
Comment (mattclark-zerogravit) [reply app/services/mapping/declaration/GoodsItemQuantityBuilder.scala]: ExportsItemsCacheIds was a class inside itemscachingService. In reality this reference is going to be deleted as the cachemap / exportItems reference will cease to exist once the old caching is full removed. Decided to do this as a quick fix so the code compiles.
